index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>TIC TAC TOE</title>
+<link rel="stylesheet" href="style.css">
+<script src="script.js"></script>
+</head>
+
+<body>
+
+<div id="board">
+	<div class="sq r1 c1 b1" onclick="play(this)"></div>
```

Comment:
You can remove the `onclick` in the markup and just loop over the `.sq` elements inside your JS file to add the listener.